src/Routes.js

```diff
@@ -1,17 +1,23 @@
 import React from 'react';
-import Main from './Pages/Main/Main';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-
+import Main from './Pages/Main/Main';
+import Category from './Pages/Category/Category';
+import Login from './Pages/Login/Login';
+import Signup from './Pages/Login/Signup';
+import Product from './Pages/Product/Product';
 class Routes extends React.Component {
```

Comment:
```suggestion
import Product from './Pages/Product/Product';

class Routes extends React.Component {
```
구분되는 요소들끼리는 한줄 띄워주는게 좋겠습니다